src/electron/electron/context_menu.cljs

```diff
@@ -28,7 +28,7 @@
             (when-let [misspelled-word (not-empty (.-misspelledWord params))]
               (. menu append
                  (MenuItem. (clj->js {:label
-                                      "Add to dictionary"
```

Comment:
Unfortunately we can't translate files under src/electron yet as the user's preferred language is only accessible on the client side of the app e.g. src/main/frontend. I'll undo these for now but feel free to create an issue if you'd like to make them translatable later